python/ray/actor.py

```diff
@@ -341,10 +342,14 @@ def _remote(self,
                 *copy.deepcopy(args), **copy.deepcopy(kwargs))
         else:
             # Export the actor.
-            if not self._exported:
+            if (self._last_export_session is None
+                    or self._last_export_session < worker._session_index):
+                # If this actor class was exported in a previous session, we
+                # need to export this function again, because current GCS
+                # doesn't have it.
+                self._last_export_session = worker._session_index
```

Comment:
This more or less mimics https://github.com/ray-project/ray/blob/1a39fee9c6fa0f81e1184182b47ef106269613fe/python/ray/remote_function.py#L112-L116